src/AutorestSwift/Code Generator/AutoRestHandler.swift

```diff
@@ -3,35 +3,51 @@ import NIO
 import os.log
 
 class AutoRestHandler {
-    private let logger: Logger
+    // MARK: Properties
 
-    init() {
-        self.logger = Logger(withFileName: "autorest-swift-debug.log")
+    private let plugin: AutoRestPlugin
+    private var logger: Logger {
+        return plugin.logger
```

Comment:
This was to eliminate duplication, since the handler can reach back to its parent plugin.